app/server/methods/userMethods.js

```diff
@@ -0,0 +1,38 @@
+Meteor.methods({
+
+  setupUser: function(userId) {
```

Comment:
This is only for user signups as opposed to admin created users. The name of the method should reflect this